src/runtime/replaceNode.js

```diff
@@ -19,6 +19,9 @@ export default (oldNodes, newNodes) => {
 	oldNodes = toArray(oldNodes);
 	newNodes = toArray(newNodes);
 	const parentNode = oldNodes[0].parentNode;
+	if (!parentNode) {
```

Comment:
This seem to be bug somewhere else. `replaceNode` should not be called on detached nodes.